nginx/conf.d/ssl/ssl.vhost-zed.conf.j2

```diff
@@ -2,6 +2,54 @@ upstream backend-zed {
         server unix:/tmp/.fpm.zed.sock;
 }
 
+server {
+    listen 8080;
+##    server_name ~^zed\\.de\\..+\\.local$;
+    server_name {{ZED_HOST}};
+    keepalive_timeout 0;
+    access_log /dev/stdout extended;
+    {% if ZED_HTTP_TO_HTTPS is defined %}{% if ZED_HTTP_TO_HTTPS == "1" %}
```

Comment:
Do we really need the first if (`if ZED_HTTP_TO_HTTPS is defined`)? As I understand the config will not added if the env vars ZED_HTTP_TO_HTTPS is not defined.

Reply:
Yes, it is required since if ZED_HTTP_TO_HTTPS is not defined, then it can't compare it's value